📣add🆕favorite icon 💻/📱

mariobodemann opened the pull request:
...addressing #5 ... and having fun with PR descriptions. 😂

mariobodemann committed:
add favorite icon

cketti commented:
In my opinion the logo doesn't work very well as favicon. Maybe we should go for a simpler image?
At the very least we should get rid of the blank border.

mariobodemann committed:
pushing trimmed icons

mariobodemann commented:
I ✂ the borders. I'll think about 🖼🎨 some more ...

mariobodemann committed:
use only head of voltron as favicon

mariobodemann committed:
add custom tool for recration of favicons

The old way of creating icons used a white background, which is not always desireable.

mariobodemann committed:
adding missed favicons

cketti commented on tools/generate_favicons.sh:
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Please exevute me from the img folder to regenerate the favicons.
```
exevute -> execute

mariobodemann committed:
typo in generator

cketti commented:
lgtm
